src/regions/ca/info/ca-nt/reporting.md

```diff
@@ -1,11 +1,6 @@
 ## Northwest Territories: What should I do if I have symptoms of COVID-19?
 
-You should get tested if you are experiencing symptoms of COVID-19:
-
-- and you have travelled outside of the NWT; OR
-- you have been in close contact with someone who is symptomatic and has travelled; OR
-- you have been in close contact with a person who has been diagnosed with or is under investigation for COVID-19, OR
-- lab exposure to biological material if you feel you meet this criteria, please contact your public health unit or health centre for further information as described under [How do I get tested for COVID-19](https://www.hss.gov.nt.ca/en/services/coronavirus-disease-covid-19/getting-tested-covid-19#how-do-i-get-tested-for-covid-19-)
+Anyone experiencing symptoms of COVID-19 should contact a healthcare provider. They may recommend that you get tested.
```

Comment:
Who is "they" in this context?

Reply:
"they" refers to the healthcare provider they should contact if COVID-19 symptoms are experienced.